sdk/appconfiguration/app-configuration/src/generated/src/appConfigurationContext.ts

```diff
@@ -1,51 +1,63 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
  *
  * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
 import * as coreHttp from "@azure/core-http";
-import * as Models from "./models";
+import { ApiVersion10, AppConfigurationOptionalParams } from "./models";
+
+const packageName = "app-configuration";
+const packageVersion = "1.1.1";
 
 export class AppConfigurationContext extends coreHttp.ServiceClient {
+  endpoint: string;
   syncToken?: string;
-  apiVersion: string;
-  credentials: coreHttp.TokenCredential | coreHttp.ServiceClientCredentials;
+  apiVersion: ApiVersion10;
 
   /**
    * Initializes a new instance of the AppConfigurationContext class.
-   * @param apiVersion The API version to be used with the HTTP request.
    * @param credentials Subscription credentials which uniquely identify client subscription.
-   * @param [options] The parameter options
+   * @param endpoint The endpoint of the App Configuration instance to send requests to.
+   * @param apiVersion Api Version
+   * @param options The parameter options
    */
   constructor(
     credentials: coreHttp.TokenCredential | coreHttp.ServiceClientCredentials,
-    apiVersion: string,
-    options?: Models.AppConfigurationOptions
+    endpoint: string,
+    apiVersion: ApiVersion10,
+    options?: AppConfigurationOptionalParams
   ) {
-    if (apiVersion == undefined) {
-      throw new Error("'apiVersion' cannot be null.");
+    if (credentials === undefined) {
+      throw new Error("'credentials' cannot be null");
+    }
+    if (endpoint === undefined) {
+      throw new Error("'endpoint' cannot be null");
     }
-    if (credentials == undefined) {
-      throw new Error("'credentials' cannot be null.");
+    if (apiVersion === undefined) {
+      throw new Error("'apiVersion' cannot be null");
     }
 
+    // Initializing default values for options
     if (!options) {
       options = {};
     }
 
+    if (!options.userAgent) {
+      const defaultUserAgent = coreHttp.getDefaultUserAgentValue();
```

Comment:
Basically this branch should never be used since the package version (today) comes from a separate spot. 

If we can get the auto-version-updater code to update it in this file instead that would also be fine.

Reply:
I think the comment above addresses this.

Reply:
It does partially - the concern I have is that the auto-version updater is what ensures that we're passing the right package version in the user agent string. If updating to this latest generated client breaks that then we need to fix that before we commit this PR.

I don't think it's a huge fix though but I just want to make sure we don't end up with this not being accurate.

Reply:
Same as above. Updated the configuration for Version Updater Tool at https://github.com/Azure/azure-sdk-for-js/pull/13095/files#diff-a2820367f311281552f23cbd1d81de6e2e015b7f0d001a1d4118452b44ee9652R82